src/DatePicker/datePickerHeader.css

```diff
@@ -1,35 +1,68 @@
-@import '../proto/definitions/_colors.css';
-@import '../proto/definitions/_fonts.css';
+@import '../proto/definitions/variables.css';
 
 .default
 {
-    position:               relative;
-    display:                flex;
-    justify-content:        center;
-    align-items:            center;
+    position:            relative;
 
-    margin-bottom:          2rem;
+    // margin-bottom:       2rem;
+    padding-top:         var( --spacing-3 );
+    padding-right:       var( --spacing-3 );
+    padding-bottom:      var( --spacing-2 );
+    padding-left:        var( --spacing-3 );
+
+    background-color:    var( --PC-BLUE );
 
     .date
     {
-        @extend %Nessie-Bold;
+        @extend %Nessie;
+        color:          var( --PC-WHITE );
+        font-size:      var( --typo-7 );
+        font-weight:    300;
+        line-height:    var( --line-height-m );
+
         margin-bottom:  0;
+        margin-top:     var( --spacing-2 );
+        padding-top:    var( --spacing-3 );
+        position:       relative;
+
+        display:        inline-block;
+        width:          50%;
     }
 
-    .prev,
-    .next
+    .year
     {
-        position:   absolute;
-        top:        calc( 50% - 0.8rem);
+        font-size:      1.3rem;
+        font-weight:    400;
+        line-height:    var( --line-height-s );
+
+        position:       absolute;
+        top:            0;
+        left:           0;
     }
 
-    .prev
+    .buttonsWrapper
     {
-        left:   0;
+        display: flex;
+        flex-flow: row nowrap;
+        justify-content: space-between;
+        align-items: center;
+        align-content: center;
```

Comment:
Please indent the property values here, i.e:
```
display:         flex;
flex-flow:       row nowrap;
justify-content: space-between;
align-items:     center;
align-content:   center;
```

Also I think use of `flex-flow` is unnecessary here, since the default for `display: flex` is `flex-direction: row` and `flex-wrap: nowrap`.